### ISSUE TITLE
DropoutWrapper in rnn_cell

### ISSUE THREAD

**Issue by caiqi** [tensorflow/tensorflow#8381]
DropoutWrapper in rnn_cell do not have a state for training or testing.  Although the keep_prob can be passed as a tensor conditioned on training/testing, would it be possible to add a state argument like in  tf.nn.dropout?

**Comment by drpngx** [comment]
Thanks for your interest @caiqi !

This is a question best asked on stackoverflow, where we monitor all issues with the tag `tensorflow`.

We keep this channel for actual tensorflow bugs.
